### PR TITLE
iPXE building report

### PR DESCRIPTION
**Pull request by kotylamichal**
Notes with instructions how to build and run iPXE on APU2

### CODE REVIEW

File: dev-notes/ipxe-apu2.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# iPXE on APU2
```
Comment (pietrushnic) [inline dev-notes/ipxe-apu2.md:1]:
```suggestion
# iPXE on PC Engines apu2
```

File: dev-notes/ipxe-apu2.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+## Prerequisites
+- APU2 with serial connection to host PC
```
Comment (pietrushnic) [inline dev-notes/ipxe-apu2.md:4]:
```suggestion
- PC Engines apu2 with serial connection to host PC
```

File: dev-notes/ipxe-apu2.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+We are looking for `ipxe.lkrn`.
+
+## Booting to MBOOT2 iPXE
+
+Enter to the `bin` folder and run simple HTTP file server:
+
+`$ python3 -m http.server`
+
+Power on APU2 device and enter to the stock iPXE shell by typing `Ctrl+B`. Inside
```
Comment (pietrushnic) [inline dev-notes/ipxe-apu2.md:63]:
```suggestion
Power on PC Engines apu2 device and enter to the stock iPXE shell by typing `Ctrl+B`. Inside
```

**Review by pietrushnic — APPROVED** [review]

File: dev-notes/ipxe-apu2.md (hunk 2 above)
Comment (kotylamichal) [reply dev-notes/ipxe-apu2.md:4]: applied

File: dev-notes/ipxe-apu2.md (hunk 3 above)
Comment (kotylamichal) [reply dev-notes/ipxe-apu2.md:63]: applied

**Pull request merged**